__tests__/uitls.test.ts

```diff
@@ -0,0 +1,20 @@
+import { MstError } from "../src/utils"
```

Comment:
**issue:** @JuanJo4 - would you mind renaming this file to `__tests__/utils.test.ts`?  I think the file name is a typo currently.

Reply:
oh! my bad... fixed.